sunpy/__init__.py

```diff
@@ -13,18 +13,13 @@
 
 from sunpy.tests.runner import SunPyTestRunner
 from sunpy.util import system_info
-from sunpy.util.config import load_config, print_config
```

Comment:
why? This is a thing we provide in the base namespace so people can see what config they are running.

Reply:
It isn't in the all at least.

Reply:
hmm, fair, but it will still be there if you do:
```python
import sunpy
sunpy.print_config
```
so removing it is still a change. I think we should keep it, and maybe add it to the `__all__`.

Reply:
Sure, we can't hide it but I feel like it should either be removed or in the all.

Reply:
Fixed.